src/pages/ConvertVrt/Convert/index.tsx

```diff
@@ -1,141 +1,218 @@
+            <Token symbol={VRT_ID} css={styles.successModalToken} variant="small2" />
+            <Typography variant="small2" css={[styles.fontWeight600, styles.successMessage]}>
+              {convertWeiToCoins({
+                valueWei: vrtAmountWei,
+                tokenId: VRT_ID,
+                returnInReadableFormat: true,
+              })}
+            </Typography>
+            <Icon name="arrowShaft" css={styles.successModalArrow} />
+            <Token symbol={XVS_ID} css={styles.successModalToken} variant="small2" />{' '}
+            <Typography variant="small2" css={[styles.fontWeight600, styles.successMessage]}>
+              {xvsAmountWei &&
+                convertWeiToCoins({
+                  valueWei: xvsAmountWei,
+                  tokenId: XVS_ID,
+                  returnInReadableFormat: true,
+                })}
+            </Typography>
+          </div>
+        ),
+      });
+    } catch (err) {
+      toast.error({ title: (err as Error).message });
+    }
+  };
+  const userVrtBalance =
+    userVrtBalanceWei &&
+    convertWeiToCoins({ valueWei: userVrtBalanceWei, tokenId: VRT_ID }).toFixed();
   return (
     <div css={styles.root}>
       {walletConnected ? (
         <section css={styles.title}>
-          <Typography variant="h3">{readableXvsAvailable}</Typography>
+          <Typography variant="h3">{readableXvsAvailable || PLACEHOLDER_KEY}</Typography>
           <Typography variant="small2">{t('convertVrt.xvsAVailable')}</Typography>
         </section>
       ) : (
         <div css={styles.smallSpacer} />
       )}
       <ConnectWallet message={t('convertVrt.connectWalletToConvertVrtToXvs')}>
-        <AmountForm onSubmit={noop} maxAmount={vrtLimitWei.toFixed()} css={styles.form}>
-          {({ values }) => {
-            const xvsValue = values.amount
-              ? new BigNumber(values.amount).times(xvsToVrtRate).dp(VRT_DECIMAL).toFixed()
-              : '';
-            return (
-              <>
-                <div css={styles.inputSection}>
-                  <Typography variant="small2" css={styles.inputLabel}>
-                    {t('convertVrt.convertVrt')}
-                  </Typography>
-                  <FormikTokenTextField
-                    tokenId={VRT_ID}
-                    name="amount"
-                    css={styles.input}
-                    description={
-                      <Trans
-                        i18nKey="convertVrt.balance"
-                        components={{
-                          White: <span css={styles.whiteLabel} />,
-                        }}
-                        values={{ amount: format(new BigNumber(vrtLimitWei)) }}
-                      />
+        <EnableToken
+          title={t('convertVrt.enableVrt')}
+          assetId={VRT_ID}
+          isEnabled={userVrtEnabled}
+          tokenInfo={[
+            {
+              iconName: 'xvs',
+              label: 'VRT Conversion Ratio',
```

Comment:
This needs to be internationalized

Reply:
I am going to remove this. I added this before the designs were complete for this view.